spec/lib/open_food_network/group_buy_report_spec.rb

```diff
@@ -56,8 +56,8 @@ module OpenFoodNetwork
 
       line_items = @orders.map(&:line_items).flatten.select{ |li| li.product.supplier == @supplier1 && li.variant == @variant1 }
 
-      sum_quantities = line_items.map(&:quantity).sum
-      sum_max_quantities = line_items.map { |li| li.max_quantity || 0 }.sum
+      sum_quantities = line_items.map(&:quantity).to_a.sum
+      sum_max_quantities = line_items.map { |li| li.max_quantity || 0 }.to_a.sum
```

Comment:
Do we need to add the `to_a` here, if we're not passing a block?

Reply:
this `to_a` seems pointless to me. `map` returns an array already.